include/mc_rtc/Configuration.h

```diff
@@ -1355,6 +1355,48 @@ struct MC_RTC_UTILS_DLLAPI ConfigurationArrayIterator
 template<>
 void MC_RTC_UTILS_DLLAPI Configuration::operator()(const std::string & key, std::string & v) const;
 
+/*! \brief \ref Configuration object that keeps track of the file it comes from
+ *
+ * This utility saves you from keeping track of the loaded file, i.e. insteaf of
+ *
+ * ```cpp
+ * std::string path = "...";
+ * mc_rtc::Configuration config(path);
+ * // Work with config
+ * config.save(path);
+ * ```
+ *
+ * You can simply do:
+ *
+ * ```cpp
+ * mc_rtc::Configuration config(path);
```

Comment:
`mc_rtc::ConfigurationFile config(path);`

Reply:
Thanks, I've fixed it in d669c8a